katas/content/distinguishing_states/plus_minus/solution.md

```diff
@@ -0,0 +1,14 @@
+Both input states are superposition states, with equal absolute values of amplitudes of both basis states. This means if the sate is measured in the Pauli $Z$ basis, there is a 50-50 chance of measuring `One` or `Zero`, which won't give us the necessary information.
+
+To determine in which state the input qubit is with certainty, we want to transform the qubit into a state where there is no superposition with respect to the basis in which we perform the measurement.
+
+Consider how we can prepare the input states, starting with basis states: $H\ket{0} = \ket{+}$ and $H\ket{1} = \ket{-}$. This transformation can also be undone by applying the **H** gate again (remember that the **H** gate is self-adjoint, i.e., it equals its own inverse): $H\ket{+} = \ket{0}$ and $H\ket{-} = \ket{1}$.
```

Comment:
```suggestion
Consider how we can prepare the input states, starting with basis states: $H\ket{0} = \ket{+}$ and $H\ket{1} = \ket{-}$. This transformation can also be undone by applying the $H$ gate again (remember that the $H$ gate is self-adjoint, i.e., it equals its own inverse): $H\ket{+} = \ket{0}$ and $H\ket{-} = \ket{1}$.
```

Minor suggestion, we enclose gates in dollar signs for latex highlighting.

Reply:
Thank you for the suggestion! If I look into the workbook, H gate is bolded rather. @tcNickolas, could you please confirm which one to use?

Reply:
The $H$ one. We had a lot of different styles for highlighting gates, I'm trying to unify them, and making them into formulas is a good way to do that, since this way they look the same in formulas and in text.